ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/TransactionComponents/SendTransactionButton.qml

```diff
@@ -19,19 +20,51 @@ Item {
         font.weight: Font.Medium
         anchors.right: parent.right
         anchors.left: parent.left
-        bottomPadding: Style.current.halfPadding
         topPadding: Style.current.halfPadding
-        anchors.top: separator1.bottom
+        anchors.top: separator.bottom
         font.pixelSize: 15
 
         MouseArea {
             anchors.fill: parent
             cursorShape: Qt.PointingHandCursor
             onClicked: {
-                console.log('Sign')
+                walletModel.setFocusedAccountByAddress(commandParametersObject.fromAddress)
+                var acc = walletModel.focusedAccount
```

Comment:
Why do we need to set a focused account? Is there possibly a way to simply retrieve the account by address from nim without needing to track an account?

Reply:
It was possible, up until the point where I needed the `assets` of that account. There is currently no way to get a `QVariant` return type with a parameter. So I couldn't get the assets by `address`. It's kinda sucky